modules/cli/src/main/scala/scala/cli/commands/package0/Package.scala

```diff
@@ -708,12 +718,41 @@ object Package extends ScalaCommand[PackageOptions] with BuildCommandHelpers {
       .withOsKind(Properties.isWin)
       .callsItself(Properties.isWin)
       .withJavaOpts(build.options.javaOptions.javaOpts.toSeq.map(_.value.value))
-    val params = Parameters.Bootstrap(Seq(loaderContent), mainClass)
+    var params = Parameters.Bootstrap(Seq(loaderContent), mainClass)
```

Comment:
IMO, `var` is not required in this code snippet.

Reply:
Yep, that's an oversight.